panel/tests/widgets/test_terminal.py

```diff
@@ -0,0 +1,60 @@
+import panel as pn
+import sys
+import subprocess
+import shlex
+
+pn.config.js_files["xtermjs"]="https://unpkg.com/xterm@4.11.0/lib/xterm.js"
+pn.config.css_files.append("https://unpkg.com/xterm@4.11.0/css/xterm.css")
```

Comment:
Assume this is for testing? These should be declared on the model.

Reply:
Sure. One step at a time :-)